config/routes.rb

```diff
@@ -18,6 +18,7 @@
 
   resources :nodes, only: [:index, :show, :destroy] do
     collection do
+      put :update
```

Comment:
for now it's not a single resource that we're patching, remember we're sending a payload with multiple node assignments, although it could be done like that too, but routing it then to the `assign_role` instance method. but maybe that's sth for later